oneanddone/tests/functional/README.md

```diff
@@ -0,0 +1,93 @@
+[mailing_list]:https://mail.mozilla.org/listinfo/mozwebqa
+
+How to run the tests locally
+-----------------------------------------
+We maintain a [detailed guide][runningtests] to running our automated tests which can be found on the MDN website.
+If you want to get started quickly, you can try following the steps in our quick-start instructions below:
+
+###Clone the repository
+If you have cloned this project already then you can skip this, otherwise you'll need to clone this repo using Git.
+If you do not know how to clone a GitHub repository, check out this 
+[help page] (https://help.github.com/articles/cloning-a-repository/) from GitHub.
+
+If you think you would like to contribute to the tests by writing or maintaining them in the future,
+it would be a good idea to create a fork of this repository first, and then clone that.
+GitHub also has great instructions for [forking a repository] (https://help.github.com/articles/fork-a-repo/).
+
+###Create or activate a Python virtual environment
+You should install this project's dependencies (which is described in the next step) into a virtual environment
+in order to avoid impacting the rest of your system, and to make problem solving easier.
+If you already have a virtual environment for these tests, then you should activate it, 
+otherwise you should create a new one.
+For more information on working with virtual environments see our 
+[our quickstart guide] (https://wiki.mozilla.org/QA/Execution/Web_Testing/Automation/Virtual_Environments) 
+and also [this blog post] (http://www.silverwareconsulting.com/index.cfm/2012/7/24/Getting-Started-with-virtualenv-and-virtualenvwrapper-in-Python).
+
+### Install dependencies
+Install the Python packages that are needed to run our tests using pip. In a terminal, 
+from the the project root, issue the following command:
+
+    pip install -Ur requirements/test.txt
+
+### Run the tests
+
+Tests are run using the command line. Below are a couple of examples of running the tests:
+
+To run all of the desktop tests against the development environment, 
+pass in the correct value for --baseurl, like so:
+
+	py.test --baseurl=https://oneanddone-dev.allizom.org --driver=firefox oneanddone/tests/functional/
+	
+You can also run the tests against a local instance of One and Done, if you have the project
+properly configured. To do so, the command line would be:
+
+	py.test  --driver=firefox oneanddone/tests/functional/
+
+Note that in order to run the tests locally you also need to install the local test dependencies into
+your virtualenv, like so:
+
+    pip install -Ur requirements/test-local.txt
+
+The pytest plugin that we use for running tests has a number of advanced command-line 
+options available. To see the options available, try running:
+
+    py.test --help
+
+The full documentation for the plugin can be found at the [pytest-selenium GitHub project page] [pytestselenium].
+[pytestselenium]: https://github.com/davehunt/pytest-selenium
```

Comment:
In recent READMEs we've gathered the links at the bottom, which I find easier to maintain and makes the raw document easier to read. Just a suggestion though, no need to change if you prefer this way. See https://raw.githubusercontent.com/mozilla/sumo-tests/master/README.md for an example.